src/main/kotlin/uk/gov/dluhc/printapi/service/FilenameFactory.kt

```diff
@@ -14,6 +16,10 @@ class FilenameFactory(private val clock: Clock) {
 
     private fun createFilename(batchId: String, count: Int, ext: String): String {
         val timestamp = LocalDateTime.now(clock).toInstant(ZoneOffset.UTC)
-        return "$batchId-$timestamp-$count.$ext"
+        return "$batchId-${TIMESTAMP_FORMATTER.withZone(ZoneId.of("UTC")).format(timestamp)}-$count.$ext"
```

Comment:
For readability purposes, it will be nice to evaluate this `${TIMESTAMP_FORMATTER.withZone(ZoneId.of("UTC")).format(timestamp)}` beforehand please